### PR TITLE
update ipv6 integration check to create & test managed nodegroup

### PR DESCRIPTION
**Pull request by aclevername**
### Description

```
[0] • [SLOW TEST:26.294 seconds]
[0] (Integration) [EKS IPv6 test]
[0] /home/jake/weave/eksctl/integration/tests/ipv6/ipv6_test.go:50
[0]   Creating a cluster with IPv6
[0]   /home/jake/weave/eksctl/integration/tests/ipv6/ipv6_test.go:55
[0]     should support ipv6
[0]     /home/jake/weave/eksctl/integration/tests/ipv6/ipv6_test.go:117
...
1 Passed | 0 Failed | 0 Pending | 0 Skipped
```

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: integration/tests/ipv6/ipv6_test.go [hunk 1]
```diff
@@ -171,6 +191,29 @@ var _ = Describe("(Integration) [EKS IPv6 test]", func() {
...
+			for _, pod := range pods.Items {
+				Expect(pod.Namespace).To(Equal(test.Namespace))
+
+				req := test.PodProxyGet(&pod, "", "/version")
+				fmt.Fprintf(GinkgoWriter, "url = %#v", req.URL())
+
+				var js interface{}
+				test.PodProxyGetJSON(&pod, "", "/version", &js)
+
+				Expect(js.(map[string]interface{})).To(HaveKeyWithValue("version", "1.5.1"))
```
Comment (nikimanoledaki) [inline integration/tests/ipv6/ipv6_test.go:215]: hm how come we're hardcoding `1.5.1`?

Comment (Skarlso) [reply integration/tests/ipv6/ipv6_test.go:215]: It's the version coming from `data/podinfo.yaml`:

```
      - name: podinfod
        image: stefanprodan/podinfo:1.5.1@sha256:702633d438950f3675d0763a4ca6cfcf21a4d065cd7f470446c67607b1a26750
```

Comment (nikimanoledaki) [reply integration/tests/ipv6/ipv6_test.go:215]: ah yep, thanks! :)

File: integration/tests/ipv6/ipv6_test.go [hunk 2]
```diff
@@ -52,6 +55,11 @@ var _ = Describe("(Integration) [EKS IPv6 test]", func() {
...
 
 		BeforeSuite(func() {
+			f, err := ioutil.TempFile("", "kubeconfig-")
+			Expect(err).NotTo(HaveOccurred())
+			params.KubeconfigPath = f.Name()
+			params.KubeconfigTemp = true
+
 			clusterConfig = api.NewClusterConfig()
 			clusterConfig.Metadata.Name = clusterName
 			clusterConfig.Metadata.Version = "1.21"
```
Comment (nikimanoledaki) [inline integration/tests/ipv6/ipv6_test.go:65]: this is a bit out of scope but would you mind making the addons use `latest` as part of this PR? I'm happy to add this commit!

Comment (aclevername) [reply integration/tests/ipv6/ipv6_test.go:65]: go for it!

Comment (aclevername) [reply integration/tests/ipv6/ipv6_test.go:65]: feel free to merge the PR afterwards

Comment (nikimanoledaki) [reply integration/tests/ipv6/ipv6_test.go:65]: done! :) thanks

**Review by nikimanoledaki — APPROVED** [review]

**Review by Skarlso — APPROVED** [review]

**Pull request merged**